targets/TARGET_Cypress/TARGET_PSOC6/TARGET_WHD/resources/firmware/TARGET_43012/resources.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019, Cypress Semiconductor Corporation, All Rights Reserved
+ * SPDX-License-Identifier: LicenseRef-PBL
+ *
+ * This file and the related binary are licensed under the
+ * Permissive Binary License, Version 1.0 (the "License");
+ * you may not use these files except in compliance with the License.
+ *
+ * You may obtain a copy of the License here:
+ * LICENSE-permissive-binary-license-1.0.txt and at
+ * https://www.mbed.com/licenses/PBL-1.0
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* Automatically generated file - this comment ensures resources.h file creation */
+/* Auto-generated header file. Do not edit */
+#ifndef INCLUDED_RESOURCES_H_
+#define INCLUDED_RESOURCES_H_
+#include "wiced_resource.h"
+
+extern const resource_hnd_t wifi_firmware_image;
+extern const char wifi_firmware_image_data[360823];
```

Comment:
This look like a really big value, is this intentional. What is this magic number anyway?

Reply:
@AnttiKauppila: this is generated file, as well as the firmware image itself - [43012C0_bin.c](https://github.com/ARMmbed/mbed-os/pull/10692/files/d77fb26e9f2d2f67fd3c6802ead4915e4dc11642#diff-1ef39d778285ff9127a03db0d408ae5f) - it generated from the binary blob that is loaded to the CYW43012 WiFi Radio: [43012C0.bin](https://github.com/ARMmbed/mbed-os/pull/10692/files/d77fb26e9f2d2f67fd3c6802ead4915e4dc11642#diff-9c4b8b71db880f2b410b723ec317768a).